### PR TITLE
updating images in distro in robotello properties sample

### PR DESCRIPTION
**Pull request by tstrych**

### CODE REVIEW

File: robottelo.properties.sample [hunk 1]
```diff
@@ -187,8 +187,8 @@ ssh_key=
 
 # For tests that uses the images for content-host testcases.
 # [distro]
-# image_el6=rhel68
-# image_el7=rhel73
+# image_el6=rhel610
```
Comment (vijay8451) [inline robottelo.properties.sample]: Not requires to update this one as RHEL6.10 is latests one ..

File: robottelo.properties.sample [hunk 2]
```diff
@@ -187,8 +187,8 @@ ssh_key=
...
+# image_el7=rhel76
```
Comment (vijay8451) [inline robottelo.properties.sample]:
```suggestion
# image_el7=rhel77
```
RHEL7.7 is latest GA here ..

File: robottelo.properties.sample (hunk 1 above)
Comment (tstrych) [reply robottelo.properties.sample]: @vijay8451 Can you please explain to me, why we do not need to change this ? I am probably missing the idea which version should be in distro section.

Comment (vijay8451) [reply robottelo.properties.sample]: ahh .. you can ignore ... I missed the version change rhel610 :)

**Pull request merged**